src/Arduino_ConnectionHandler.h

```diff
@@ -113,6 +110,19 @@ class ConnectionHandler {
   #define NETWORK_CONNECTED GSM3_NetworkStatus_t::GPRS_READY
 #endif
 
+#ifdef ARDUINO_SAMD_MKRGSM1400
```

Comment:
Why is the code part `ARDUINO_SAMD_MKRGSM1400` added again? It already exists in the file starting from line 108.

Reply:
Fixed